addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -63,6 +63,14 @@
     column-gap: 0px !important;
 }
 
+// Adapt the horizontal margins of a direct row child of a grid item, to make
+// them compensate the grid item horizontal padding (to avoid an overflow).
+.o_grid_item > .row {
+    --grid-inner-row-gutter-x: clamp(0px, 2 * var(--grid-item-padding-x), 30px);
+    margin-left: calc(-0.5 * var(--grid-inner-row-gutter-x));
+    margin-right: calc(-0.5 * var(--grid-inner-row-gutter-x));
+}
+
```

Comment:
Shouldn't we just override the `--gutter-x` variable for that direct row? And make it clamp? What's the risk?

Reply:
That's what I tried to do first, but the issue is that `--gutter-x` also impacts the padding of the columns inside this row (they use the same variable), so I didn't want to change the way the columns themselves look (in mobile view for example, they would look different from the normal mode ones) :grimacing:
So, since the overflow issue is really only because of the margins, I preferred only fixing them :slightly_smiling_face: 

Note that in 17.0, this rule will change to target the first row inside a grid item, so not necessarily a direct child (because of a gutter rule that changed in mobile view in 17.0) :+1: (still to check)

Reply:
By clamping the `--gutter-x`, with a grid padding of 0px: 
![image](https://github.com/user-attachments/assets/1e1efb9c-add4-4a14-9cf3-5be81e2e9d82)

In normal mode:
![image](https://github.com/user-attachments/assets/672c3587-e0ca-4df4-8e52-9b2693eb6724)

With my solution:
![image](https://github.com/user-attachments/assets/d07034ef-f346-4da8-a4f7-32609f2b78aa)